Stack.py

```diff
@@ -0,0 +1,122 @@
+#pila:
+pila = []
+
```

Comment:
python ya tiene si lectura de pilas, pero te hizo falta crear una operacion de lectura
```suggestion
print(pila)
```